apps/tailwind-components/components/Pagination.vue

```diff
@@ -64,12 +84,26 @@ function changeCurrentPage(event: Event) {
     <span :id="`${pageInputId}Label`" class="sr-only">
       pagination navigation
     </span>
-    <ul class="flex items-center justify-center list-none">
+    <ul class="flex items-center justify-center list-none gap-[10px]">
+      <li v-if="jumpToEdge">
+        <a
```

Comment:
More of a general question: how do we handle the state of an element if it's at the min/max value (e.g., first page, last page, etc.). Should we disable the element?